replit_river/session.py

```diff
@@ -128,6 +120,7 @@ async def _begin_close_session_countdown(self) -> None:
         self._close_session_after_time_secs = (
             await self._get_current_time() + grace_period_ms / 1000
         )
+        await self.close_websocket(self._ws_wrapper, should_retry=not self._is_server)
```

Comment:
we shouldn't retry when we disconnect due to session timeout

Reply:
this it the begin of session timeout, starting the countdown, we should retry here right?

Reply:
oh wait you are right yes, and its fine that this potentially races with another reconnect?

Reply:
the reconnects are all controlled by the lock in transport, so there will be only one at a time